modules/transformations/redshift/lib/helper.py

```diff
@@ -103,3 +105,45 @@ def calculate_final_bearing(start, end):
     bear = bearing(end, start)
     bear = (bear + 180) % 360
     return bear
+
+
+def geom_from_geojson(geom):
```

Comment:
I would rename this functions to avoid confusions `geom_from_geojson`-> `wkt_from_geojson`.